SimpleWAWS/Code/TemplatesManager.cs

```diff
@@ -60,35 +64,72 @@ static TemplatesManager()
                     Name = "Ping Site",
                     SpriteName = "sprite-PingSite PingSite",
                     AppService = AppService.Logic,
-                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/CSMTemplates/PingSite.json"),
-                    Description = Resources.Server.Templates_PingSiteDescription
+                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/ARMTemplates/PingSite.json"),
+                    Description = Resources.Server.Templates_PingSiteDescription,
+                    MSDeployPackageUrl = $"https://github.com/fashaikh/appservice-zipped-templates/raw/master/Logic/Ping%20Site.zip"
                 });
                 list.Add(new JenkinsTemplate
                 {
                     Name = "Jenkins CI",
                     SpriteName = "sprite-PingSite PingSite",
                     AppService = AppService.Jenkins,
-                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/CSMTemplates/JenkinsResource.json"),
+                    CsmTemplateFilePath = HostingEnvironment.MapPath("~/ARMTemplates/JenkinsResource.json"),
                     Description = Resources.Server.Templates_JenkinsDescription
                 });
+                _baseARMTemplate = JObject.Parse(File.ReadAllText(HostingEnvironment.MapPath("~/ARMTemplates/BaseARMTemplate.json"),  System.Text.Encoding.ASCII)
+                    .Replace(Environment.NewLine, String.Empty)
+                    .Replace(@"\", String.Empty));
                 //TODO: Implement a FileSystemWatcher for changes in the directory
             }
-            catch (Exception)
+            catch (Exception ex)
             {
                 _templatesList = Enumerable.Empty<BaseTemplate>();
             }
         }
 
+        private static object getShortName(string templateName)
```

Comment:
it's more of a css name or css sprite name than a general shortName. I think this method name is a bit confusing. Also it should be PascalCase